app/src/main/cpp/skyline/services/nvdrv/devices/nvhost/gpu_channel.cpp

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: MIT OR MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#include <soc.h>
+#include <services/nvdrv/devices/deserialisation/deserialisation.h>
+#include "gpu_channel.h"
+
+namespace skyline::service::nvdrv::device::nvhost {
+    GpuChannel::GpuChannel(const DeviceState &state, Core &core, const SessionContext &ctx) :
+        NvDevice(state, core, ctx),
+        smExceptionBreakpointIntReportEvent(std::make_shared<type::KEvent>(state, false)),
+        smExceptionBreakpointPauseReportEvent(std::make_shared<type::KEvent>(state, false)),
+        errorNotifierEvent(std::make_shared<type::KEvent>(state, false)) {
+        channelSyncpoint = core.syncpointManager.AllocateSyncpoint(false);
+    }
+
+    PosixResult GpuChannel::SetNvmapFd(In<core::NvMap::Handle::Id> id) {
+        state.logger->Debug("id: {}", id);
```

Comment:
Same as before, ID should be capitalized, there's other instances of this that you can probably search up so I'll stop pointing it out.

Reply:
I would rather not here, it's printing the variable name as per the convention of the other nvdrv logs

Reply:
I suppose it's fine then